libproxy/cmake/devfiles.cmk

```diff
@@ -1,4 +1,4 @@
-if(NOT WIN32)
+if(NOT WIN32 OR MINGW)
```

Comment:
For completeness the related endif should also contain the condition; can you please fix that?